src/data_type/age.verification.js

```diff
@@ -1,24 +1,30 @@
 'use strict';
 
 const Validation = require('../yoti_common/validation');
+const constants = require('../yoti_common/constants');
 
 /**
- * Wraps an 'Age Verify/Condition' attribute to provide behaviour specific
+ * Based on an 'Age Verify/Condition' attribute name and value, provides behaviour specific
  * to verifying someone's age.
  *
  * @class AgeVerification
  */
 class AgeVerification {
-  constructor(attribute) {
-    Validation.notNull(attribute, 'attribute');
-    Validation.matchesPattern(attribute.getName(), /^[^:]+:(?!.*:)[0-9]+$/, 'attribute.name');
-    this.attribute = attribute;
+  static isAttributeNameMatchingAgeVerification(name) {
+    return name.startsWith(constants.ATTR_AGE_OVER)
+      || name.startsWith(constants.ATTR_AGE_UNDER);
+  }
+
+  constructor(name, value) {
+    Validation.isString(name, 'name');
+    Validation.oneOf(value, ['true', 'false'], 'value');
```

Comment:
Shouldn't the constructor be written first?

Reply:
I don't think there is any rules around that, I could not find anything after 5min of googling, feel free to provide some documentation if you have.
I think this is only a matter of style, I am used to see them on top, as the first thing a class offers, before reading what is related to the instance behaviour (getter/setter/constructor/instance methods).

Reply:
Yes, I couldn't find any doc or article about this.
But there are two other classes in the node SDK which write the constructor before static.

Reply:
Ok, so let's go with that.
We may follow stricter convention when we go Typescript.